homeassistant/components/squeezebox/media_player.py

```diff
@@ -241,14 +220,16 @@ class SqueezeBoxEntity(MediaPlayerEntity):
     _last_update: datetime | None = None
     _attr_available = True
 
-    def __init__(self, player):
+    def __init__(self, player: Player) -> None:
         """Initialize the SqueezeBox device."""
         self._player = player
-        self._query_result = {}
-        self._remove_dispatcher = None
+        self._query_result: bool | dict = {}
+        self._remove_dispatcher: None | callable = None
```

Comment:
```suggestion
        self._remove_dispatcher: Callable | None = None
```
Don't forget to add the import